.saucetpl/.sauce/config.yml

```diff
@@ -17,6 +17,7 @@ suites:
   - name: "saucy test"
     platformName: "Windows 10"
     testMatch: '**/*.js'
+      #screenResolution: "1920x1080"  # Available resolutions on sauce for Windows: '800x600', '1024x768', '1152x864', '1280x768', '1280x800', '1280x960', '1280x1024', '1400x1050', '1440x900', '1600x1200', '1680x1050', '1920x1080', '1920x1200', '2560x1600'
```

Comment:
Since it's commented out , and since `saucectl` is reading & saving the yaml configuration file when fetching the template, the will just disappear and never be show to the user typing `saucectl new`

Reply:
True, though it's fine to include this though. Later we have to fix the templating anyway, because it's not acceptable that comments disappear. They are especially very useful when starting out with saucectl.

Reply:
😂 updating